server/src/internalClusterTest/java/org/opensearch/discovery/StableClusterManagerDisruptionIT.java

```diff
@@ -135,14 +135,14 @@ private void ensureNoMaster(String node) throws Exception {
     }
 
     /**
-     * Verify that nodes fault detection detects a disconnected node after master reelection
+     * Verify that nodes fault detection detects a disconnected node after cluster-manager reelection
      */
     public void testFollowerCheckerDetectsDisconnectedNodeAfterMasterReelection() throws Exception {
         testFollowerCheckerAfterMasterReelection(NetworkDisruption.DISCONNECT, Settings.EMPTY);
     }
 
     /**
-     * Verify that nodes fault detection detects an unresponsive node after master reelection
+     * Verify that nodes fault detection detects an unresponsive node after cluster-manager reelection
      */
     public void testFollowerCheckerDetectsUnresponsiveNodeAfterMasterReelection() throws Exception {
```

Comment:
`testFollowerCheckerDetectsUnresponsiveNodeAfterMasterReelection` -> `testFollowerCheckerDetectsUnresponsiveNodeAfterClusterManagerReelection`?

Reply:
Changed the method name in the new commit https://github.com/opensearch-project/OpenSearch/pull/2521/commits/02af408bcf8bf72ebff9f3a01592cf3cf0e9f49f.